dist/api_data.py

```diff
@@ -594,7 +594,10 @@ def __cmp__(self, other):
         Config('recover', 'on', r'''
             run recovery or error if recovery needs to run after an
             unclean shutdown''',
-            choices=['error','on'])
+            choices=['error','on']),
+        Config('recover_progress', 'true', r'''
+            print messages indicating progress of recovery''',
+            type='boolean'),
```

Comment:
Using `wt_msg` means I had to make this configurable in order to make it quiet in order for the tests to pass.